src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -340,4 +340,13 @@ describe('ReactIncrementalScheduling', () => {
     // animation priority.
     expect(ReactNoop.getChildren()).toEqual([span(1)]);
   });
+
+  it('can force synchronous updates with syncUpdates, even inside batchedUpdates', () => {
+    ReactNoop.batchedUpdates(() => {
+      ReactNoop.syncUpdates(() => {
+        ReactNoop.render(<span />);
+        expect(ReactNoop.getChildren()).toEqual([span()]);
```

Comment:
If `batchedUpdates` didn't immediately execute the provided callback, the test would complete without any assertions (since it's not an async test) which could potentially give you a false positive if there was a failing assertion inside of the inner callback.